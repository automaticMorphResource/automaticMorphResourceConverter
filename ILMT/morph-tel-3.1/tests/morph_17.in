1	varRA	unk	
1	kAlaM	unk	
1	vacciMxi	unk
1	aMte	unk	
1	palleVllo	unk	
1	puttagoVdugulu	unk	
1	eVkkadapadiwe	unk	
1	akkada	unk	
1	kanipiswAyi	unk	
1	.	unk	
