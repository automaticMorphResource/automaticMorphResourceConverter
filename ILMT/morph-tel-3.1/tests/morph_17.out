1	varRA	unk	<fs af='varRA,n,,sg,,d,0,0'>|<fs af='varRA,n,,sg,,o,ti,ti'>
2	kAlaM	unk	<fs af='kAlaM,n,,sg,,d,0,0'>|<fs af='kAlu,v,,pl,1,,a,a'>
3	vacciMxi	unk	<fs af='vaccu,v,fn,sg,3,,A,A'>
4	aMte	unk	<fs af='aMte,avy,,,,,,'>|<fs af='aMta,avy,,,,,e,e_avy'>|<fs af='aMtu,n,,sg,,,0,0_e'>|<fs af='aMtu,v,any,any,any,,e,e'>|<fs af='aMtu,v,any,any,any,,i,i_e'>|<fs af='anu,v,any,any,any,,we,we'>
5	palleVllo	unk	<fs af='palleV,n,,pl,,,lo,lo'>
6	puttagoVdugulu	unk	<fs af='puttagoVdugu,n,,pl,,d,0,0'>|<fs af='puttagoVdugu,n,,pl,,,vu,vu'>
7	eVkkadapadiwe	unk	<fs af='eVkkadapadiwe,avy,,,,,,'>
8	akkada	unk	<fs af='akkada,adv,,,,,0,0_adv'>
9	kanipiswAyi	unk	<fs af='kanipiMcu,v,n,pl,3,,wA,wA'>
10	.	unk	<fs af='&dot;,punc,,,,,,'>
